### ISSUE TITLE
Search widget: opening search is not announced as a new route

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#19892]
When opening the search (click on the magnifying glass in the search demo of gallery) the search page is not announced as a new route with its title.

**Comment by goderbauer** [comment]
This is also a problem on iOS

**Comment by jonahwilliams** [comment]
Fixed by https://github.com/flutter/flutter/pull/20190
